scripts/publish.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+# Publish an updated version of pileup.js on NPM.
+
+set +x
+set -o errexit
+
+# Ensure a clean dist directory
+rm -rf dist
+
+npm run jstransform
+npm run browserify
+npm run uglify
+
+# jstransform creates this, but we don't want to distribute it.
+rm -rf dist/.module-cache
+
+# No need to distribute tests
+rm -rf dist/test
```

Comment:
That doesn't get generated because I run `npm run browserify`, rather than `npm run browserify-all`.